R/deployApp.R

```diff
@@ -48,6 +48,22 @@
 #' @param appTitle Free-form descriptive title of application. Optional; if
 #'   supplied, will often be displayed in favor of the name. If ommitted,
 #'   on second and subsequent deploys, the title will be unchanged.
+#' @param envVars A character vector giving the names of environment variables
+#'   whose values should be synchronised with the server (currently supported by
+#'   Connect only). The values of the environment variables are sent over an
+#'   encrypted connection and are not stored in the bundle, making this a safe
+#'   way to send private data to Connect.
```

Comment:
We should eventually create example workflows:

1. named environment variables that are assigned on each deploy.
2. named environment variables that are assigned only on first deploy.
3. modifying the set of environment variables over time (across multiple deploys).